Bottom: da4342e8e6d562fbc741e1ecafbbdf6a3674d7fd
Top:    5e9c600a5ef88b004993ed010f09308a49ed8780
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-23 21:38:18 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 0ab30b0..8821559 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1807,7 +1807,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.6
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
